### ISSUE TITLE
关于dropout的报错

### ISSUE THREAD

**Issue by anlan-he** [doc-ycm/EEG-ATCNet-pytorch#2]
您好，在forward中，遇到问题
att_out = self.drop(att_out)
报错为dropout(): argument 'input' (position 1) must be Tensor, not tuple
请问这里应该怎么修改呢？
尝试了att_out = torch.tensor([item.cpu().detach().numpy() for item in att_out]).cuda()，但是说维度不对

**Comment by Wangzilng** [comment]
MHA返回的是
output, attn_scores
所以是个元组，所以我感觉应该把return的attn_scores删了，好像看起来没用到

**Comment by MiaoJasmin** [comment]
现在怎么样了 这个为什么准确率没有tf高呢
